### PR TITLE
Flutter

### PR DESCRIPTION
**Pull request by albertopasqualetto**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

Install gave error from my tests because of malwares found (but windows defender does not find any).
Not sure if everything is correct.
This solves https://github.com/microsoft/winget-pkgs/issues/95594, https://github.com/microsoft/winget-pkgs/issues/18414 and https://github.com/flutter/flutter/issues/57674
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/126336)

### CODE REVIEW

**Comment by albertopasqualetto** [comment]
@microsoft-github-policy-service agree

**Comment by albertopasqualetto** [comment]
I think I've done what was necessary, I do not know why validation is failing

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335141

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 2023-11-22 10:45:51.436 [CLI ] Unable to locate nested installer at: C:\Users\User\AppData\Local\Temp\WinGet\Google.Flutter.3.13.9\extracted\flutter\bin\dart.exe 

Apparently the file doesn't have an EXE extension in the ZIP folder. 

image

File: manifests/g/Google/Flutter/3.13.9/Google.Flutter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+PackageVersion: "3.13.9"
+InstallerType: zip
+ReleaseDate: 2023-10-26
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: flutter\bin\flutter.bat
+    PortableCommandAlias: flutter
+  - RelativeFilePath: flutter\bin\dart.exe
```
Comment (stephengillie) [inline manifests/g/Google/Flutter/3.13.9/Google.Flutter.installer.yaml:11]:
```suggestion
  - RelativeFilePath: flutter\bin\dart.bat
```

**Comment by albertopasqualetto** [comment]
Well, that was obvious... sorry. Didn't notice that.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> An error occurred trying to start process 'C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\flutter.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.

**Review by stephengillie — APPROVED** [review]

**Comment by albertopasqualetto** [comment]
> what is the status of this package? It produces a not 64bit package error for flutter when run.

I'am not an expert, but there is no error on my side with `flutter create my_app` and `flutter run` for browser.

**Comment by mwisnicki** [comment]
Seems like this was deleted in https://github.com/microsoft/winget-pkgs/pull/131091

**Pull request merged**